static/css/style.css

```diff
@@ -184,41 +211,97 @@ a:hover {
     cursor: text;
 }
 
-.rating {
-    color: #ffaa1d;
+/* Image and image selector */
+
+.overlay-container {
+    position: relative;
+    width: 100%;
+}
+
+.card-img {
+    display: block;
+    width: 100%;
+    height: auto;
+}
+
+.image-overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 100%;
+    width: 100%;
+    opacity: 0;
+    transition: 0.5s ease;
+    background-color: #000;
+    cursor: pointer;
+    z-index: 2;
+}
+
+.overlay-container:hover .image-overlay {
+    opacity: 0.5;
+}
+
+.overlay-text {
+    color: white;
     font-size: 1.5rem;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
```

Comment:
You can safely drop these prefixed declarations. 2D CSS transforms are supported by 97%+ of all browsers: https://caniuse.com/#feat=transforms2d.